service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/PlacementSubmittedControllerTest.java

```diff
@@ -79,20 +80,67 @@ void shouldSendNotificationWhenAddingNewChildPlacement() throws Exception {
 
         postSubmittedEvent(callbackRequest);
 
+        verify(notificationClient).sendEmail(
+            NEW_PLACEMENT_APPLICATION_NOTIFICATION_TEMPLATE, "admin@family-court.com",
+            expectedTemplateParameters(), String.valueOf(CASE_ID));
+
+        verify(coreCaseDataService).triggerEvent(JURISDICTION, CASE_TYPE_ID, CASE_ID, SEND_DOCUMENT_EVENT, Map.of(
+            "documentToBeSent", documentReference
+        ));
+
+        verify(notificationClient, never()).sendEmail(
+            NEW_PLACEMENT_APPLICATION_NOTIFICATION_TEMPLATE, "FamilyPublicLaw+ctsc@gmail.com",
+            expectedTemplateParameters(), String.valueOf(CASE_ID));
+
         verify(notificationClient).sendEmail(NEW_PLACEMENT_APPLICATION_NOTIFICATION_TEMPLATE,
             "admin@family-court.com",
             expectedTemplateParameters(),
             String.valueOf(CASE_ID));
     }
 
+    @Test
+    void shouldNotifyCtscAdminWhenAddingNewChildPlacementAndCtscIsEnabled() throws Exception {
+        Element<Child> child1 = testChild();
+        Element<Child> child2 = testChild();
+
+        DocumentReference child1Application = testDocument();
+        DocumentReference child2Application = testDocument();
+
+        Element<Placement> child1Placement = element(testPlacement(child1, child1Application));
+        Element<Placement> child2Placement = element(testPlacement(child2, child2Application));
+
+        CallbackRequest callbackRequest = CallbackRequest.builder()
+            .caseDetails(CaseDetails.builder()
+                .id(CASE_ID)
+                .data(ImmutableMap.<String, Object>builder()
+                    .putAll(buildNotificationData())
+                    .putAll(buildPlacementData(List.of(child1, child2), List.of(child2Placement, child1Placement),
+                        child2.getId()))
+                    .put("sendToCtsc", "Yes")
+                    .build())
+                .build())
+            .caseDetailsBefore(CaseDetails.builder().data(new HashMap<>()).build())
+            .build();
```

Comment:
this could be extracted

Reply:
Refactored to make this cleaner.

Reply:
I was expecting this to move to private method